cmd/mkcgo/main.go

```diff
@@ -16,6 +16,7 @@ var (
 	fileName      = flag.String("out", "", "output file name (standard output if omitted)")
 	includeHeader = flag.String("include", "", "include header file")
 	packageName   = flag.String("package", "", "package name")
+	private       = flag.Bool("private", false, "all Go generated symbols are kept unexported")
```

Comment:
Is this exercised? Maybe at least have a test case to make sure the feature keeps working. (And maybe document when it'd be used?)

Reply:
I use this for other experiments. Plan to add some test cases in a follow-up PR.